src/kOS/Suffixed/Addons.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using kOS.Safe.Encapsulation.Suffixes;
+using kOS.Safe.Encapsulation;
+
+namespace kOS.Suffixed
+{
+    public class AddonList : Structure
+    {
+        private Addon KACAddon;

```

Comment:
This is a place where two standards are clashing.  Don't use lowercase for acronyms is another common practice with good reason to exist.  (i.e. call it "NASA" not "nasa")